src/main/java/org/testeditor/fixture/commons/text/NameGenerator.java

```diff
@@ -105,7 +105,10 @@ public JsonObject getPerson() throws FixtureException {
             person.addProperty("userName", nextPerson.getUserName());
             person.addProperty("companyName", nextPerson.getCompany());
             person.addProperty("domainName", nextPerson.getDomainName());
-            person.addProperty("email", nextPerson.getEmail());   
+            person.addProperty("email", nextPerson.getEmail());
+            PersonAsString personAsString = new PersonAsString();
+            personAsString.setPerson(nextPerson);
+            logger.debug("Got following person data : " + personAsString.toString());
```

Comment:
I'd rather add a private method to the fixture that does a toString(Person) instead of adding a class 'PersonAsString' which essentially contains only this one method.

Reply:
I made a Pull-Request at the original project [Inbot Pull-Request](https://github.com/Inbot/inbot-testfixtures/pull/13). We can wait until the Pull-Request is merged into the master branch, so we can use it in a new release of the Inbot project. But i think we can use this part until then, what do you think :O)